tests/test_worktree_management.py

```
@@ -191,3 +191,128 @@ def test_list_worktrees_empty_when_none_exist(tmp_path, git_env):
         assert res.stdout.strip() == ""  # Empty output
     finally:
         os.chdir(original_dir)
+
+
+def _init_bare_repo(bare_path: Path) -> None:
+    """Initialize a bare repository."""
+    subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)